modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -239,6 +246,47 @@ definition {
 		return ${response};
 	}
 
+	macro createStudentRelatedToAccount {
```

Comment:
Just FYI, I'm not sure if we use the createCustomObjectRelatedToSystemObject is better than createStudentRelatedToAccount

Reply:
the custom object is not parametrised in this method, it is exactly students, and it is exactly account as a system object. If we want to name it as you suggest we would need to parametrise it, and it may not be worth it.